modules/force_rtl.py

```diff
@@ -2,23 +2,22 @@
 Forces drone to return to launch (RTL).
 """
 
-import dronekit
+# import dronekit
+from modules.common.mavlink.modules.flight_controller import FlightController
 
 from . import upload_commands
 from . import generate_command
 
-MAVLINK_RTL_FRAME = dronekit.mavutil.mavlink.MAV_FRAME_GLOBAL
-MAVLINK_RTL_COMMAND = dronekit.mavutil.mavlink.MAV_CMD_NAV_RETURN_TO_LAUNCH
 DRONE_TIMEOUT = 30.0  # seconds
 
 
-def force_rtl(drone: dronekit.Vehicle) -> bool:
+def force_rtl(drone: FlightController) -> bool:
```

Comment:
maybe rename drone as controller, makes it clear that it is NOT the dronekit.vehicle